src/OmnichannelChatSDK.ts

```diff
@@ -779,7 +780,7 @@ class OmnichannelChatSDK {
         }
     }
 
-    public async endChat(): Promise<void> {
+    public async endChat(optionalParams?: EndChatOptionalParams): Promise<void> {
```

Comment:
What would happen of `chatSDK.endChat()` was called?

Should it have `optionsParams` set to `{}` by default to be backward compatible?